src/content/app/genome-browser/components/drawer/drawer-views/variant-group-legend/VariantGroupLegendColours.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export default [
+  {
+    id: 1,
+    colour: '#E685AE'
+  },
+  { id: 2, colour: '#F8C041' },
+  {
+    id: 3,
+    colour: '#84FA3A'
+  },
+  {
+    id: 4,
+    colour: '#327C89'
+  },
+  {
+    id: 5,
+    colour: '#96D0C9'
+  }
+];
```

Comment:
Something to think about — can we add the new colours to our colour palette, and use them from there.

E.g. what if we generate classNames in `VariantGroupsLegend` component based on color identifiers (e.g. `.variantColor1`, etc.), which will then use the colors defined out our color palette for background color?